const e=JSON.parse(`{"key":"v-aeadea66","path":"/%E7%BC%96%E7%A8%8B%E8%AF%AD%E8%A8%80/python/%E6%B5%81%E7%A8%8B%E6%8E%A7%E5%88%B6/traceback.html","title":"traceback","lang":"zh-CN","frontmatter":{"description":"traceback python会在抛出exceptions时打印traceback, 来帮助你查找出错的地方. 创建以下代码, 可以看到形参是someone, 而在函数里面, 我们写成了someon, 所以在调用的时候会报错. def greet(someone): print('Hello, ' + someon) greet('Chad')","head":[["meta",{"property":"og:url","content":"https://claroja.github.io/%E7%BC%96%E7%A8%8B%E8%AF%AD%E8%A8%80/python/%E6%B5%81%E7%A8%8B%E6%8E%A7%E5%88%B6/traceback.html"}],["meta",{"property":"og:site_name","content":"Claroja"}],["meta",{"property":"og:title","content":"traceback"}],["meta",{"property":"og:description","content":"traceback python会在抛出exceptions时打印traceback, 来帮助你查找出错的地方. 创建以下代码, 可以看到形参是someone, 而在函数里面, 我们写成了someon, 所以在调用的时候会报错. def greet(someone): print('Hello, ' + someon) greet('Chad')"}],["meta",{"property":"og:type","content":"article"}],["meta",{"property":"og:locale","content":"zh-CN"}],["meta",{"property":"og:updated_time","content":"2023-11-27T13:17:01.000Z"}],["meta",{"property":"article:author","content":"claroja"}],["meta",{"property":"article:modified_time","content":"2023-11-27T13:17:01.000Z"}],["script",{"type":"application/ld+json"},"{\\"@context\\":\\"https://schema.org\\",\\"@type\\":\\"Article\\",\\"headline\\":\\"traceback\\",\\"image\\":[\\"\\"],\\"dateModified\\":\\"2023-11-27T13:17:01.000Z\\",\\"author\\":[{\\"@type\\":\\"Person\\",\\"name\\":\\"claroja\\",\\"url\\":\\"https://claroja.github.io\\"}]}"]]},"headers":[{"level":2,"title":"Common Tracebacks in Python?","slug":"common-tracebacks-in-python","link":"#common-tracebacks-in-python","children":[{"level":3,"title":"AttributeError","slug":"attributeerror","link":"#attributeerror","children":[]},{"level":3,"title":"ImportError","slug":"importerror","link":"#importerror","children":[]},{"level":3,"title":"IndexError","slug":"indexerror","link":"#indexerror","children":[]},{"level":3,"title":"KeyError","slug":"keyerror","link":"#keyerror","children":[]},{"level":3,"title":"NameError","slug":"nameerror","link":"#nameerror","children":[]},{"level":3,"title":"SyntaxError","slug":"syntaxerror","link":"#syntaxerror","children":[]},{"level":3,"title":"TypeError","slug":"typeerror","link":"#typeerror","children":[]},{"level":3,"title":"ValueError","slug":"valueerror","link":"#valueerror","children":[]}]}],"git":{"createdTime":1701091021000,"updatedTime":1701091021000,"contributors":[{"name":"claroja","email":"63183535@qq.com","commits":1}]},"readingTime":{"minutes":2.51,"words":752},"filePathRelative":"编程语言/python/流程控制/traceback.md","localizedDate":"2023年11月27日","excerpt":"<h1> traceback</h1>\\n<p><code>python</code>会在抛出<code>exceptions</code>时打印<code>traceback</code>, 来帮助你查找出错的地方.</p>\\n<p>创建以下代码, 可以看到形参是<code>someone</code>, 而在函数里面, 我们写成了<code>someon</code>, 所以在调用的时候会报错.</p>\\n<div class=\\"language-python line-numbers-mode\\" data-ext=\\"py\\"><pre class=\\"language-python\\"><code><span class=\\"token keyword\\">def</span> <span class=\\"token function\\">greet</span><span class=\\"token punctuation\\">(</span>someone<span class=\\"token punctuation\\">)</span><span class=\\"token punctuation\\">:</span>\\n    <span class=\\"token keyword\\">print</span><span class=\\"token punctuation\\">(</span><span class=\\"token string\\">'Hello, '</span> <span class=\\"token operator\\">+</span> someon<span class=\\"token punctuation\\">)</span>\\n\\ngreet<span class=\\"token punctuation\\">(</span><span class=\\"token string\\">'Chad'</span><span class=\\"token punctuation\\">)</span>\\n</code></pre><div class=\\"line-numbers\\" aria-hidden=\\"true\\"><div class=\\"line-number\\"></div><div class=\\"line-number\\"></div><div class=\\"line-number\\"></div><div class=\\"line-number\\"></div></div></div>","copyright":{"author":"王新宇"},"autoDesc":true}`);export{e as data};
